### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

**Comment by themak1985** [comment]
@trancer1994  https://videony.rhsj520.com/0803/AVOP-447/1500kb/hls/index.m3u8 is dead.

**Comment by LaneSh4d0w** [comment]
> 
> 
> @trancer1994 https://videony.rhsj520.com/0803/AVOP-447/1500kb/hls/index.m3u8 is dead.

Not 24/7 it seems. Works for me.

**Comment by LaneSh4d0w** [comment]
Wait...

https://videony.rhsj520.com/0803/AVOP-447/1500kb/hls/index.m3u8

That isn't a channel... that's a 4 hour video !

**Review by heroku-miraheze — APPROVED** [review]
Works fully.

**Comment by freearhey** [comment]
https://videony.rhsj520.com/0803/AVOP-447/1500kb/hls/index.m3u8 doesn't work for me either

**Comment by LaneSh4d0w** [comment]
This link seems to work in an anarchic way, at least it works for me, but tends to lag.

Nevertheless, it isn't a channel at all.

File: channels/us.m3u [hunk 1]
```diff
@@ -3554,3 +3554,11 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/Cj6kg2X.png" group-title="",CBTV Now
+https://oj7lng29dg82-hls-live.5centscdn.com/lives/f7b44cfafd5c52223d5498196c8a2e7b.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",AVOP-447
+https://videony.rhsj520.com/0803/AVOP-447/1500kb/hls/index.m3u8
```
Comment (freearhey) [inline channels/us.m3u:3560]: https://github.com/iptv-org/iptv/pull/2921#issuecomment-850063476
https://github.com/iptv-org/iptv/pull/2921#issuecomment-850200322

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by heroku-miraheze** [comment]
It's not 24/7 stream.

**Comment by LaneSh4d0w** [comment]
> 
> 
> It's not 24/7 stream.

It's not because it's a 24/7 stream, it's because it's not a channel at all.

**Comment by themak1985** [comment]
> > It's not 24/7 stream.
> 
> It's not because it's a 24/7 stream, it's because it's not a channel at all.

Agreed, this is not a real channel.

**Comment by themak1985** [comment]
Fixed merge issue and removed AVOP-447 - I think this is good t go now after someone else reviews.

**Review by freearhey — APPROVED** [review]

**Pull request merged**